statistics/handle/cache/statscache.go

```diff
@@ -67,11 +67,9 @@ func (s *StatsCache) UpdateCache(newCache StatsCacheWrapper) (updated bool, newC
 	s.mu.Lock()
 	oldCache := s.cache.Load()
 	newCost = newCache.Cost()
-	if oldCache.version < newCache.version || (oldCache.version == newCache.version && oldCache.minorVersion < newCache.minorVersion) {
-		s.memTracker.Consume(newCost - oldCache.Cost())
-		s.cache.Store(&newCache)
-		updated = true
-	}
+	s.memTracker.Consume(newCost - oldCache.Cost())
```

Comment:
Maybe we can still keep the comparison of version since `(s *StatsCache).UpdateCache` can be called concurrently.

Reply:
1) even if we keep the comparison, there are still some bad cases that can cause unexpected overwriting; 2) the side-effect of overwriting here seems trivial and acceptable; So to make it easier to understand and maintain, maybe removing the version comparison here is better.